config/crd/bases/controlplane.cluster.x-k8s.io_awsmanagedcontrolplanes.yaml

```diff
@@ -808,6 +808,8 @@ spec:
                       description: ControlPlane determines if this failure domain
                         is suitable for use by control plane machines.
                       type: boolean
+                  required:
```

Comment:
Trying to figure out why the CRD generation makes this a required field, which should not be the case.